Makefile

```diff
@@ -202,8 +204,10 @@ install-cfg: docs/stratisd.8
 install: $(PROFILETARGET) install-cfg
 	install -Dpm0755 -t $(DESTDIR)$(LIBEXECDIR) target/$(PROFILEDIR)/stratisd
 	install -Dpm0755 -t $(DESTDIR)$(UDEVDIR) target/$(PROFILEDIR)/stratis-utils
-	mv $(DESTDIR)$(UDEVDIR)/stratis-utils $(DESTDIR)$(UDEVDIR)/stratis-str-cmp
-	ln $(DESTDIR)$(UDEVDIR)/stratis-str-cmp $(DESTDIR)$(UDEVDIR)/stratis-base32-decode
+	mv -fv $(DESTDIR)$(UDEVDIR)/stratis-utils $(DESTDIR)$(UDEVDIR)/stratis-str-cmp
+	ln -fv $(DESTDIR)$(UDEVDIR)/stratis-str-cmp $(DESTDIR)$(UDEVDIR)/stratis-base32-decode
+	mkdir -p $(DESTDIR)$(BINDIR)
```

Comment:
Can you explain this line? I don't think we should be creating standard directories for users and if they are installing into a non-standard directory, my thought is that it should already be there.

Reply:
You're right about that. I was a little too focused on getting it working in our Jenkins tests, but for real use we'ld better drop it.